security.rst

```diff
@@ -2654,7 +2654,7 @@ Secondly, you can use the special "attribute" ``IS_AUTHENTICATED_FULLY`` in plac
 
     public function adminDashboard(): Response
     {
-        $this->denyAccessUnlessGranted('IS_AUTHENTICATED');
+        $this->denyAccessUnlessGranted('IS_AUTHENTICATED_FULLY');
```

Comment:
looking at the previous paragraph it's rather looks like we should change the sentence before instead to refer to `IS_AUTHENTICATED`

Reply:
I'm sorry but I don't understand...

Reply:
There is a paragraph before this code block which should be changed instead of changing the code example.

Reply:
Thanks for your reply! 
I wasn't sure between IS_AUTHENTICATED and IS_AUTHENTICATED_FULLY.